nbconvert/filters/markdown_mistune.py

```diff
@@ -53,8 +84,17 @@ class MarkdownWithMath(mistune.Markdown):
     def __init__(self, renderer, **kwargs):
         if 'inline' not in kwargs:
             kwargs['inline'] = MathInlineLexer
+        if 'block' not in kwargs:
+            kwargs['block'] = MathBlockLexer
         super(MarkdownWithMath, self).__init__(renderer, **kwargs)
 
+    def output_block_math(self):
```

Comment:
This should have its name changed to map the new name (e.g., `output_multiline_math`).